modules/net-vpc/variables.tf

```diff
@@ -227,6 +227,19 @@ variable "subnets" {
   nullable = false
 }
 
+variable "subnets_global_proxy_only" {
```

Comment:
Do we really need a new variable for this? I'd rather reuse the existing `subnets_proxy_only` and add a new `global` attribute with type `optional(boolean, FALSE)`.

Reply:
That's fair - I'll have a read how to implement optional into the existing proxy subnet.

Reply:
@juliocc  - i've refactored to use the existing proxy_only subnet, and removed all the other work done to reflect. Introduced test to create the additional subnet and regenerated the doco